content/en/blog/_posts/2022-11-11-retroactive-default-storage-class.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Retroactive Default StorageClass"
+date: 2022-12-06
```

Comment:
Do we have a date set? Or was this supposed to go out yesterday? Please amend the dates here per the 1.26 sheet.

Reply:
Wasn't sure which date is this supposed to represent, changing this to "Release blog published" date.

Reply:
I've also changed the name of the file to match this one.